terasoluna-gfw-functionaltest-web/src/main/resources/META-INF/spring/applicationContext.xml

```diff
@@ -103,4 +103,6 @@
         <property name="exceptionLogger" ref="variationExceptionLogger" />
     </bean>
 
+    <context:component-scan base-package="org.terasoluna.gfw.functionaltest.config" />

```

Comment:
I changed the place <context:component-scan/> close to <context:property-placeholder/> at 7d584291e90ead1fa680c17fba777c7b755237e3 commit